airflow/providers/google/cloud/hooks/tasks.py

```diff
@@ -120,20 +122,19 @@ def create_queue(
         client = self.get_conn()
 
         if queue_name:
-            full_queue_name = CloudTasksClient.queue_path(project_id, location, queue_name)
+            full_queue_name = f"projects/{project_id}/locations/{location}/queues/{queue_name}"
             if isinstance(task_queue, Queue):
                 task_queue.name = full_queue_name
             elif isinstance(task_queue, dict):
                 task_queue['name'] = full_queue_name
             else:
                 raise AirflowException('Unable to set queue_name.')
-        full_location_path = CloudTasksClient.location_path(project_id, location)
+        full_location_path = f"projects/{project_id}/locations/{location}"
         return client.create_queue(
-            parent=full_location_path,
-            queue=task_queue,
+            request={'parent': full_location_path, 'queue': task_queue},
             retry=retry,
             timeout=timeout,
-            metadata=metadata,
+            metadata=metadata or (),
```

Comment:
Not a strong opinion, but I think doing this before function invocation may increase readability, WDYT?

Reply:
I am not sure about that. For full readability this would require creating a new variable name, as modifying the parameter content may be ambiguous as some languages have special behavior. Luckily not in Python, but this can still arouse mixed feelings among polyglots, including mine. The new variable in this case would complicate the code unnecessarily.